check/README.md

```diff
@@ -0,0 +1,114 @@
+# Tool for work with string tables
+
+## Glossary
+
+- `string table` a file with translation
+- `reference table` en.txt string table.
+  English is the development language, so any changes related to text should be in that file.
+  Also, this table us used as fallback for other translations, if a needed key is missing.
+  This file is mostly maintained by people who change game code,
+  so the list of entries in this file should be the most up-to-date.
+- `entry` one piece of translation, it consists of
+  - `key` a sting that represents entry in the codebase (C++, Python).
+    This string must be unique.
+  - `value` actual translation that will be shown to the user.
+
+### Key
+Only uppercase Latin alphabet (A-Z) and underscore (_) are allowed.
+Renaming a key should be done with caution.
+Many of the keys in the stringtable are used in the game source code explicitly,
+or in content scripts.
+Some keys are also generated programatically, such as by adding a prefix to other text.
+C++ uses UserString(...) or UserStringNop(...) to look up strings or to mark stringtable keys that are used in code.
+Python uses `fo.userString` and `fo.userStringList` for look up.
+
+### Value syntax
+#### Simple (single line)
+A single line of text, should not be started and ended by any whitespace character.
+
+```
+KEY
+value
+
+>>>
+value
+<<<
+```
+
+#### Full (single or multiline)
+Start and end with triple single quotes.
+You can put anything except triple single quotes inside.
+
+```
+KEY_WITH_LEADING_SPACE
+''' value'''
+
+>>>
+ value
+<<<
+
+KEY_WITH_MULTILINE
+'''Hello
+world'''
+
+>>>
+Hello
+world
+<<<
+
+```
+
+
+#### Tags
+You can customize text with special markup tags. Not full list of tags:
+
+- `[[]]` inserts text from another entry.
+  You add a key in double square braces and user will see the value of that key.
+  Substitution is recursive, if text has another substitution it will be resolved too.
+
+```
+KEY_WORLD
+world
+
+KEY_HELLO
+Hello [[KEY_WORLD]]!
+
+>>>
+Hello world
+<<<
+```
+
+This feature should be used to keep the translation in a consistent state.
+You define term once and use it everywhere.
+For various kinds of game content, like techs, buildings, or species, etc., there are links
+
+- `link` inserts a link to a wiki page about some scripted content or a ship design in the game universe.
+Supported types
+  - encyclopedia
+  - buildingtype
+  - fieldtype
+  - metertype
+  - predefinedshipdesign
+  - shiphull
+  - shippart
+  - special
+  - species
+  - tech
+  - policy
```

Comment:
i'll add a new type 'auto' soon that will try to determine the appropriate matching content type from raw string, see also https://github.com/freeorion/freeorion/pull/3665/commits/88abdd6be3107669efe360a6b0fb6212308ec12c#diff-c6e7742666839efb5adf4f6a1fea0c22d9e28991d1f88c76e9062ccef54223f9